### PR TITLE
Optimize structure for quick start with tidb

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

- Fix the sidebar issue caused by wrong simpletab tags.
- Optimize structure for quick start with TiDB.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/7075
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by lilyjazz — APPROVED** [review]

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -15,11 +18,14 @@ aliases: ['/docs-cn/dev/quick-start-with-tidb/','/docs-cn/dev/how-to/get-started
 > - 如需在 Kubernetes 上部署 TiDB，请参考[快速上手 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/get-started)。
 > - 如需在云上管理 TiDB，请参考 [TiDB Cloud 快速上手指南](https://docs.pingcap.com/tidbcloud/beta/tidb-cloud-quickstart)。
 
+## 部署本地测试集群
+
+- 适用场景：利用本地 Mac 或者单机 Linux 环境快速部署 TiDB 测试集群，体验 TiDB 集群的基本架构，以及 TiDB、TiKV、PD、监控等基础组件的运行。
+- 耗时：1 分钟。
```
Comment (qiancai) [inline quick-start-with-tidb.md:24]: 这里的耗时是阅读时间吗？还是操作时间？是仅仅包括下载 tiup 以及运行 tiup playground 吗？会不会有人理解为 step 1 到 step 8 的耗时。 有的时候下载 tiup 以及运行 tiup playground 可能加起来也会超过 1 分钟。如果这个时间不太好估计，建议不写。

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -222,10 +225,13 @@ TiDB 是一个分布式系统。最基础的 TiDB 测试集群通常由 2 个 Ti
...
-## 使用 TiUP cluster 在单机上模拟生产环境部署步骤
+</div>
+</SimpleTab>
+
+## 在单机上模拟部署生产环境集群
 
-- 适用场景：希望用单台 Linux 服务器，体验 TiDB 最小的完整拓扑的集群，并模拟生产的部署步骤。
-- 耗时：10 分钟
+- 适用场景：希望用单台 Linux 服务器，体验 TiDB 最小的完整拓扑的集群，并模拟生产环境下的部署步骤。
+- 耗时：10 分钟。
```
Comment (qiancai) [inline quick-start-with-tidb.md:234]: 这个可能也要再确认下，看 10 分钟够不够

**Review by qqqdan — APPROVED** [review]

File: quick-start-with-tidb.md (hunk 1 above)
Comment (ran-huang) [reply quick-start-with-tidb.md:24]: 操作时间是初版里李仲舒老师给到的数据。下载的时间确实会超过，我删掉吧
```suggestion
```

File: quick-start-with-tidb.md (hunk 2 above)
Comment (ran-huang) [reply quick-start-with-tidb.md:233]: 感觉不太好确认，也删掉吧
```suggestion
```

Comment (lilyjazz) [reply quick-start-with-tidb.md:233]: 准备好了 topology.yaml 的话，十分钟肯定够了

File: quick-start-with-tidb.md (hunk 1 above)
Comment (lilyjazz) [reply quick-start-with-tidb.md]: 确实没包含下载时间，但是下载时间和客户网络有关，不好评估。
写成 「部署耗时」？

File: quick-start-with-tidb.md (hunk 2 above)
Comment (lilyjazz) [reply quick-start-with-tidb.md:233]: 这个也叫做 「部署耗时」？

File: quick-start-with-tidb.md (hunk 1 above)
Comment (ran-huang) [reply quick-start-with-tidb.md]: 最开始写这个时间的目的应该是降低心理门槛，鼓励用户试用，但没有这个时间也不影响操作。为了避免误解，还是删掉吧。

Comment (lilyjazz) [reply quick-start-with-tidb.md]: 听起来没问题

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**